libs/s25main/nodeObjs/noGrapefield.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2024 Settlers Freaks (sf-team at siedler25.org)
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "noCoordBase.h"
+
+class SerializedGameData;
+class GameEvent;
+
+class noGrapefield : public noCoordBase
+{
+private:
+    /// Typ of the grape field (2 different types)
```

Comment:
Better something like:
```suggestion
    /// Type of the grape field (for drawing only, currently 0-1)
```

Maybe even use an enum or bool for this (